src/main/scala/beam/agentsim/agents/choice/mode/BridgeTollDefaults.scala

```diff
@@ -44,7 +44,7 @@ object BridgeTollDefaults {
     }
   }
 
-  private def readTollPrices(tollPricesFile: String): Map[Int, Double] = {
+  def readTollPrices(tollPricesFile: String): Map[Int, Double] = {
```

Comment:
Actually I'm going to agree with @michaz and at least request this be cached in some fashion.